packages/core/src/texture/Texture.ts

```diff
@@ -52,6 +53,13 @@ export abstract class Texture extends GraphicsResource {
     return this._height;
   }
 
+  /**
+   * The texture whether used for video play.
+   */
+  get isVideo(): boolean {
```

Comment:
If you write a class that inherits, I don't recommend it! isVideo is a high-level abstraction